src/attorney.js

```diff
@@ -195,6 +196,19 @@ function applyArchiveConfig (config) {
   }
 }
 
+function applyArchiveFailedConfig (config) {
+  assert(!('archiveFailedAfterSeconds' in config) || config.archiveFailedAfterSeconds >= 1,
+    'configuration assert: archiveFailedAfterSeconds must be at least every second and less than ')
+
+  config.archiveFailedSeconds = config.archiveFailedAfterSeconds || config.archiveCompletedAfterSeconds
```

Comment:
This is failing the test suite.  `archiveCompletedAfterSeconds` is optional, so it's assigning undefined.  If `archiveFailedAfterSeconds` isn't set, we should jump to the bottom and assign from `config.archiveInterval`

Reply:
I think I fixed it by defaulting to `config.archiveSeconds` which is never undefined. 

It is strange, because when running the suite locally I didn't have the failure